src/lib/y2network/connection_config/ctc.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/connection_config/base"
+
+module Y2Network
+  module ConnectionConfig
+    # Configuration for ctc connections
+    class Ctc < Base
+      # @return [String] read bus id
+      attr_accessor :read_channel
+      # @return [String] write bus id
+      attr_accessor :write_channel
+      # @return [Integer] connection protocol
+      attr_accessor :protocol
```

Comment:
link to more details as it looks strange to have protocol identified just by number.